instrumentation/que/Appraisals

```diff
@@ -6,12 +6,19 @@
 
 appraise 'que-1.x' do
   gem 'que', '~> 1.4'
+  gem 'activerecord', '~> 7.1.0'
 end
 
 appraise 'que-2.x' do
-  gem 'que', '~> 2.3'
+  gem 'que', '~> 2.4'
+  if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('3.1.0')
+    gem 'activerecord', '~> 7.2.0'
+  else
+    gem 'activerecord'
+  end
```

Comment:
Without this, we'd skip testing on Rails 7.2 and jump straight to Rails 8.0. Que made a change to support Rails 7.2 in 2.4.1, but our instrumentation didn't need to change. 

Since these tests are for our instrumentation and not necessarily for activerecord compatibility with que, I'm not sure we need the condition. Thoughts?